pkg-install

```diff
@@ -40,6 +42,14 @@ RUN git clone https://aur.archlinux.org/yay.git
 WORKDIR /home/{user}/yay
 RUN makepkg -si --noconfirm
 
+"""
+if edit_flag is not None: #if -e flag is passed then take on the following string
+    extra_run = f"""RUN {edit_flag}"""
+else:
+    extra_run = ""
+
+dockerfile += extra_run + f"""
+
```

Comment:
There is a slightly cleaner way to do this
```python
"""
# ...
RUN makepkg -si --noconfirm
{"RUN" + edit_flag if edit_flag is not None else ""}
#...
"""

Reply:
Many thanks! the things you can do in these multi-lined strings are new to me

Reply:
Still testing, sorry!